ci/images/releasescripts/src/test/java/io/spring/concourse/releasescripts/sonatype/SonatypeServiceTests.java

```diff
@@ -105,7 +104,7 @@ void publishWithSuccessfulClose() throws IOException {
 					.filter((artifact) -> !artifact.startsWith("build-info.json"))
 					.map((artifact) -> requestTo(
 							"/service/local/staging/deployByRepositoryId/example-6789/" + artifact.toString()))
-					.collect(Collectors.toCollection(HashSet::new));
+					.collect(Collectors.toUnmodifiableSet());
```

Comment:
`Collectors.toSet()`

Reply:
We don't need mutable Set here, I think `toUnmodifiableSet()` is more appropriate

Reply:
We disagree. Please use `toSet()`.